src/app/App.spec.ts

```diff
@@ -56,17 +56,25 @@ describe('App.vue', () => {
   })
 
   test('fails to renders basic view', async () => {
+    mock('/', {}, () => {
+      return {
+        headers: {
+          'Status-Code': '500',
+        },
+        body: '',
+      }
+    })
     withVersion('10.2.0')
-    const wrapper = renderComponent('ERROR')
-
     await store.dispatch('updateGlobalLoading', true)
-    store.dispatch('bootstrap')
+    await store.dispatch('bootstrap')
+    const wrapper = renderComponent()
+
     expect(wrapper.find('[data-testid="app-progress-bar"]').exists()).toBe(true)
 
     await flushPromises()
     await store.dispatch('updateGlobalLoading', false)
     expect(wrapper.find('[data-testid="app-progress-bar"]').exists()).toBe(false)
 
-    expect(wrapper.element).toMatchSnapshot()
+    expect(wrapper.html()).toContain('Unable to reach the API')
```

Comment:
I removed the snapshot match here as the previous test setup was still loading in other API requests perfectly fine, and was creating all the donut charts etc. Really we are testing for whether the `Unable to reach the API` message is showing, so I changed it to test just for that.